src/rnp/fficli.cpp

```diff
@@ -2817,6 +2817,31 @@ cli_rnp_setup(cli_rnp_t *rnp)
     return true;
 }
 
+bool
+cli_rnp_check_weak_hash(cli_rnp_t *rnp)
+{
+    if (rnp->cfg().has(CFG_WEAK_HASH)) {
+        return true;
+    }
+
+    uint32_t security_level = 0;
+
+    if (!rnp_get_security_rule(rnp->ffi,
```

Comment:
Within this logic if `rnp_get_security_rule()` call would fail due to some reasons we'll allow to use any hash. Probbaly we should separate checks, failing (or issuing a warning) if `rnp_get_security_rule()` fails.